filebeat/harvester/log.go

```diff
@@ -292,12 +297,12 @@ func (h *Harvester) newLogFileReader() (reader.Reader, error) {
 	var r reader.Reader
 	var err error
 
-	fileReader, err := reader.NewLogFile(h.file, readerConfig, h.done)
+	logInput, err := NewLogFile(h.file, readerConfig)
```

Comment:
does `LogFile` reader close file handle? Do we have 2 code-pathes potentially closing h.file now?